libraries/AP_HAL_ChibiOS/hwdef/TBS_LUCID_PRO/hwdef.dat

```diff
@@ -0,0 +1,157 @@
+
+# hw definition file for processing by chibios_hwdef.py
+# for OMNIBUSF4 hardware.
+# thanks to betaflight for pin information
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID AP_HW_TBS_LUCID_PRO
+
+# crystal frequency, setup to use external oscillator
+OSCILLATOR_HZ 8000000
+
+FLASH_SIZE_KB 1024
+
+# bootloader takes first sector
+FLASH_RESERVE_START_KB 48
+
+define HAL_STORAGE_SIZE 16384
+define STORAGE_FLASH_PAGE 1
+
+STM32_ST_USE_TIMER 5
+
+# SPI devices
+
+# SPI1 - IMU
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+
+# SPI2 - OSD
+PB13 SPI2_SCK SPI2
+PB14 SPI2_MISO SPI2
+PB15 SPI2_MOSI SPI2
+
+# SPI3 - Flash
+PC10 SPI3_SCK SPI3
+PC11 SPI3_MISO SPI3
+PB5 SPI3_MOSI SPI3
+
+# Chip select pins
+PA15 FLASH1_CS CS
+PB12 OSD1_CS CS
+PA4 GYRO1_CS CS
+
+# Beeper
+PC13 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+
+# SERIAL ports
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6
+# PA10 IO-debug-console
+PA11 OTG_FS_DM OTG1
+PA12 OTG_FS_DP OTG1
+
+# USART1
+PA10 USART1_RX USART1
+PA9 USART1_TX USART1
+define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_RCIN
+
+# USART2 - ESC Telem
+PA2 USART2_TX USART2 NODMA
+PA3 USART2_RX USART2 NODMA
+define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_ESCTelemetry
+
+# USART3 - HD
+PB10 USART3_TX USART3
+PB11 USART3_RX USART3
+define DEFAULT_SERIAL3_PROTOCOL SerialProtocol_MSP_DisplayPort
+
+# UART4 - SmartAudio
+PA0 UART4_TX UART4 NODMA
+PA1 UART4_RX UART4 NODMA
+define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_SmartAudio
+
+# UART5 - RX, SBUS inversion
+PD2 UART5_RX UART5 NODMA
+define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_None
+
+# USART6 - GPS
+PC6 USART6_TX USART6
+PC7 USART6_RX USART6
+define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_GPS
+
+# I2C ports
+I2C_ORDER I2C1
+
+# I2C1
+PB8 I2C1_SCL I2C1
+PB9 I2C1_SDA I2C1
+
+# Servos
```

Comment:
the SPARE PC2 and PC3 solder pads are not defined in this hwdef

Reply:
Not used so not defined